Sources_v3/StreamChatUI/ChatChannel/Attachments/ChatMessageAttachmentsView.swift

```diff
@@ -69,6 +85,7 @@ open class ChatMessageAttachmentsView<ExtraData: ExtraDataTypes>: View, UIConfig
         ]
         layoutConstraints[[.images, .interactiveAttachment]] = layoutConstraints[.interactiveAttachment]
         layoutConstraints[[.files, .interactiveAttachment]] = layoutConstraints[.interactiveAttachment]
+        layoutConstraints[[.giphy, .interactiveAttachment]] = layoutConstraints[.interactiveAttachment]
```

Comment:
Shouldn't we also provide a layout for `[.giphy, .images]` options?

Reply:
Is user able to put giphy and images in one message?
I think no and in this case it is good for now. 
As we previously speak, we will rework all `LayoutOption` things soon (they not scale well). Will be action point